javascript/javascript1/week3/activity.js

```diff
@@ -0,0 +1,69 @@
+let activities = [];
+let date = new Date();
+function addActivity(date, activity, duration) {
+  activities.push({ date, activity, duration });
+}
+formattedDate =
+  date.getDate() + "-" + (date.getMonth() + 1) + "-" + date.getFullYear();
+addActivity(formattedDate, "Youtube", 30);
+addActivity(formattedDate, "facebook", 15);
+addActivity(formattedDate, "twetter", 10);
+console.log(activities);
+
+function showStatus(activities) {
+  let usageAmount = activities.reduce(function (r, a) {
+    return r + a["duration"];
+  }, 0);
```

Comment:
You haven't study `Higher-Order Functions` yet so I would like you to calculate usageAmount inside for loop
```js
  let usageAmount = 0;
  for (let i = 0; i < activities.length; i++) {
    usageAmount += activities[i].duration;
  }
  console.log(
    `You have added ${activities.length} activities. They amount to ${usageAmount} min. of usage`
  );
```